### PR TITLE
feat: add Supabase Cron

### PR DESCRIPTION
**Pull request by w3b6x9**
## I have read the [CONTRIBUTING.md](https://github.com/supabase/supabase/blob/master/CONTRIBUTING.md) file.

YES

## What kind of change does this PR introduce?

Supabase Cron

### CODE REVIEW

File: apps/www/_blog/2024-12-04-supabase-cron.mdx [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+
+<div className="video-container">
+  <iframe
+    className="w-full"
+    src="https://www.youtube-nocookie.com/embed/_fdP-aaTHgw"
+    title="We have improved Supabase AI ... A lot!"
+    allow="accelerometer; autoplay; clipboard-write; encrypted-media; fullscreen; gyroscope; picture-in-picture; web-share"
+    allowfullscreen
+  />
+</div>
```
Comment (w3b6x9) [inline apps/www/_blog/2024-12-04-supabase-cron.mdx:42]: **REMINDER** - update video

File: apps/www/data/features.tsx [hunk 2]
```diff
@@ -2041,4 +2042,29 @@ This feature is especially beneficial for Python developers looking to build rob
...
+4. Versatile Scheduling Options: Supports standard cron syntax, sub-minute intervals, and natural language scheduling.
+2. Supabase Integration: Integrates seamlessly with the rest of the Supabase platform, including Supabase Edge Functions and Database Webhooks.
+3. Zero Network Latency: Jobs are run directly in your database, eliminating network latency when executing SQL Snippets or calling Database Functions.
+6. Enhanced Observability: Monitor job history, debug errors, and review logs directly from the Supabase Dashboard.
+5. Broad Use Cases: Automate a wide range of tasks, including database maintenance, analytics, performance optimizations, and syncing with remote systems.
+7. Ease of Use: Leverage an intuitive UI or SQL for scheduling, managing, and monitoring Jobs.
+`,
+    icon: Clock,
+    products: [PRODUCT_SHORTNAMES.DATABASE, ADDITIONAL_PRODUCTS.PLATFORM],
+    heroImage: '',
```
Comment (w3b6x9) [inline apps/www/data/features.tsx:2062]: **REMINDER:** link to video

File: apps/docs/content/guides/cron/quickstart.mdx [hunk 3]
```diff
@@ -0,0 +1,329 @@
...
+<Tabs
+  scrollable
+  size="small"
+  type="underlined"
+  defaultActiveId="dashboard-edit-job"
+  queryGroup="database-method"
+>
+<TabPanel id="dashboard-edit-job" label="Dashboard">
+
+1. Go to the [Jobs](https://supabase.com/dashboard/project/_/integrations/cron/jobs) section and find the Job you'd like to edit.
```
Comment (thorwebdev) [inline apps/docs/content/guides/cron/quickstart.mdx:98]:
```suggestion
1. Go to the [Jobs](/dashboard/project/_/integrations/cron/jobs) section and find the Job you'd like to edit.
```

File: apps/www/_blog/2024-12-04-supabase-cron.mdx [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+  - postgres
+tags:
+  - cron
+  - postgres
+date: '2024-12-04'
+toc_depth: 3
+launchweek: '13'
+---
+
+Today we're releasing Supabase Cron, a new Postgres Module that makes scheduling recurring Jobs simple and intuitive inside your database.
```
Comment (kiwicopple) [inline apps/www/_blog/2024-12-04-supabase-cron.mdx:18]:
```suggestion
Today we're releasing [Supabase Cron](/modules/cron), a new Postgres Module that makes recurring Jobs simple and intuitive inside your database.
```

File: apps/www/_blog/2024-12-04-supabase-cron.mdx [hunk 5]
```diff
@@ -0,0 +1,138 @@
...
+  src="/images/blog/launch-week-13/day-3-supabase-cron/history.jpg"
+/>
+For a deeper dive, you can view Jobs in the [Logs Explorer](https://supabase.com/dashboard/project/_zvmkyvanngopzqaabmvx_/logs/pgcron-logs).
+<Img alt="View cron logs" src="/images/blog/launch-week-13/day-3-supabase-cron/logs.jpg" />
+
+### Try Supabase Cron today
+
+Getting started is easy:
+
+1. Visit the new [Integrations page](https://supabase.com/dashboard/project/_/integrations) in your project.
```
Comment (kiwicopple) [inline apps/www/_blog/2024-12-04-supabase-cron.mdx:127]:
```suggestion
1. Visit the [Integrations page](https://supabase.com/dashboard/project/_/integrations) in your project.
```

File: apps/www/data/features.tsx [hunk 6]
```diff
@@ -2036,4 +2042,29 @@ This feature is especially beneficial for Python developers looking to build rob
...
+4. Versatile Scheduling Options: Supports standard cron syntax, sub-minute intervals, and natural language scheduling.
+2. Supabase Integration: Integrates seamlessly with the rest of the Supabase platform, including Supabase Edge Functions and Database Webhooks.
+3. Zero Network Latency: Jobs are run directly in your database, eliminating network latency when executing SQL Snippets or calling Database Functions.
+6. Enhanced Observability: Monitor job history, debug errors, and review logs directly from the Supabase Dashboard.
+5. Broad Use Cases: Automate a wide range of tasks, including database maintenance, analytics, performance optimizations, and syncing with remote systems.
+7. Ease of Use: Leverage an intuitive UI or SQL for scheduling, managing, and monitoring Jobs.
+`,
+    icon: Clock,
+    products: [PRODUCT_SHORTNAMES.DATABASE, ADDITIONAL_PRODUCTS.PLATFORM],
+    heroImage: 'https://youtu.be/miRQPbIJOuQ',
```
Comment (w3b6x9) [inline apps/www/data/features.tsx:2062]: @fsansalvadore this currently breaks. how do i add yt video for this?

**Review by fsansalvadore — APPROVED** [review]

**Review by saltcod — APPROVED** [review]

**Pull request merged**